docs/changelog.md

```diff
@@ -2,6 +2,160 @@
 
 Beanie project
 
+## [1.27.0] - 2024-10-06
+### Fix return type from document update
+- Author - [CAPITAINMARVEL](https://github.com/CAPITAINMARVEL)
+- PR <https://github.com/BeanieODM/beanie/pull/1030>
+### Fix type hint using pymongo client session instead of motor client session
+- Author - [CAPITAINMARVEL](https://github.com/CAPITAINMARVEL)
+- PR <https://github.com/BeanieODM/beanie/pull/1022>
+### Add tests on all major mongo version
+- Author - [07pepa](https://github.com/07pepa)
+- PR <https://github.com/BeanieODM/beanie/pull/1034>
+### Add test to ensure dict with enum keys are encoded properly
+- Author - [adeelsohailahmed](https://github.com/adeelsohailahmed)
+- PR <https://github.com/BeanieODM/beanie/pull/1001>
+### Replace deprecated datetime.utcnow with datetime.now
+- Author - [adeelsohailahmed](https://github.com/adeelsohailahmed)
+- PR <https://github.com/BeanieODM/beanie/pull/1014>
+### Use sequence instead of list in init_beanie
+- Author - [07pepa](https://github.com/07pepa)
+- PR <https://github.com/BeanieODM/beanie/pull/1015>
+### Fix expression type hint not allowing some type https://github.com/beanieodm/beanie/issues/1020
+- Author - [CAPITAINMARVEL](https://github.com/CAPITAINMARVEL)
+- PR <https://github.com/BeanieODM/beanie/pull/1023>
+### Fix expression type hint not allowing some type https://github.com/beanieodm/beanie/issues/1020
```

Comment:
Not sure what is going on here but some entries are being repeated multiple times.
By checking the other Releases and their notes there doesn't seem to be any duplicated entries.
Perhaps the `generate_changelog.py` script should be updated to do some kind of deduplication...?

Reply:
Oof, good catch, @staticxterm!

I've removed all the duplicated entries, and ordered the change log based on the PR number for easier review. Could you please have another look?